frontend/src/components/camp/Picasso.vue

```diff
@@ -477,7 +454,6 @@ export default {
   min-width: 20px !important;
   top: 0 !important;
   right: 0 !important;
-  opacity: 0;
```

Comment:
I feel like the many small icons make the view nervous. I'd propose to only show the edit icon when hovering above a schedule entry, and hiding it always in the sidebar version. But that can easily be done in a separate (good first) issue later.

Reply:
#678